main.tf

```diff
@@ -56,7 +56,7 @@ locals {
 }
 
 data "aws_cloudwatch_event_bus" "this" {
-  count = var.create && var.create_bus ? 0 : 1
```

Comment:
this is correct as written - this is a data source. so if we are not creating the event bus, we are then looking up an existing event bus by name